Core/Object Arts/Dolphin/MVP/Type Converters/NullConverter.cls

```diff
@@ -10,6 +10,16 @@ NullConverter comment: 'NullConverter is a <typeConverter> which applies no tran
 !NullConverter categoriesForClass!MVP-Type Converters-General! !
 !NullConverter methodsFor!
 
+= comparand
```

Comment:
Seems the wrong level to introduce #hash and #=. These should be pushed up if they are needed.